### PR TITLE
Added null-check

### PR DESCRIPTION
**Pull request by Ekman1991**
Fixes lots of error messages in Sentry. Do not load the form if the user has been logged out.

### CODE REVIEW

**Comment by Ekman1991** [comment]
When the user logs out or is logged out, the event-emitter is emitting null to the user. If the user is on the profile page as he/she is logged out the profile page is trying to render/reload the site with a null user.

File: src/app/views/user/user-settings/user-profile/user-profile.component.ts [hunk 1]
```diff
@@ -129,8 +129,10 @@ export class UserProfileComponent extends TranslationListener implements OnInit
 
     this.authManager.getUserChangeEmmiter().subscribe(user => {
       this.user = user;
-      this.loadData();
-      this.initForm();
+      if(user != null) {
```
Comment (buren) [inline src/app/views/user/user-settings/user-profile/user-profile.component.ts]: `!=` => `!==`

**Review by buren — CHANGES_REQUESTED** [review]

File: src/app/views/user/user-settings/user-profile/user-profile.component.ts (hunk 1 above)
Comment (Ekman1991) [reply src/app/views/user/user-settings/user-profile/user-profile.component.ts]: Nice catch, fixed now.

**Review by buren — APPROVED** [review]

**Comment by buren** [comment]
👑

**Pull request merged**